doc/library/d3viz/index.txt

```diff
@@ -0,0 +1,274 @@
+.. _libdoc_d3viz:
+
+============================================================================
+:mod:`d3viz` -- d3viz: Interactive visualization of Theano compute graphs
+============================================================================
+
+.. module:: d3viz
+   :platform: Unix, Windows
+   :synopsis: Allows to interactively visualize Theano compute graphs
+.. moduleauthor:: Christof Angermueller
+
+
+Guide
+=====
+
+Requirements
+------------
+
+``d3viz`` requires the `pydot <https://pypi.python.org/pypi/pydot>`__
+package, which can be installed with ``pip``:
+
+::
+
+    pip install pydot
+
+Overview
+--------
+
+``d3viz`` extends Theano’s `printing
+module <http://deeplearning.net/software/theano/library/printing.html>`__
+to interactively visualize compute graphs. Instead of creating a static
+picture, it creates an HTML file, which can be opened with any
```

Comment:
Sorry to be picky about this but this should probably say "most recent [...]" instead of "any [...]". Using 'any' will lead some people to think they can just use IE6 and expect all of the Javascript and CSS to work flawlessly.